src/main/java/pl/scoutCamp/api/dto/mapper/AssignmentMapper.java

```diff
@@ -0,0 +1,11 @@
+package pl.scoutCamp.api.dto.mapper;
+
+import org.mapstruct.Mapper;
+import pl.scoutCamp.api.dto.AssignmentDTO;
+import pl.scoutCamp.domain.Assignment;
+
+@Mapper(componentModel = "spring")
+public interface AssignmentMapper {
+
+    Assignment map (AssignmentDTO assignmentDTO);
```

Comment:
Please try to avoid blank spaces before brackets. Let's stick to the code format rules